mkosi/__init__.py

```diff
@@ -6643,7 +6644,7 @@ def line_join_list(array: Sequence[PathString]) -> str:
 def print_summary(args: MkosiArgs) -> None:
     # FIXME: normal print
     MkosiPrinter.info("COMMANDS:")
-    MkosiPrinter.info("                      verb: " + args.verb)
+    MkosiPrinter.info("                      verb: " + args.verb.name)
```

Comment:
The reason you have to use `+` here, is because `__str__` is defined for `Verb`, but `__add__` is not.
```suggestion
    MkosiPrinter.info(f"                      verb: {args.verb})
```
should call `__str__` instead of `__add__` and we don't have to reach into the enum to pull out name or value.